packages/@icon-magic/distribute/src/distribute-svg.ts

```diff
@@ -72,7 +78,7 @@ export async function distributeSvg(
         icon.category && groupByCategory
           ? path.join(outputPath, icon.category)
           : outputPath;
-        await createHbs(assetsByColorScheme, destPath);
+        await createHbs(assetsByColorScheme, destPath, doNotRemoveSuffix);
```

Comment:
```suggestion
        await createHbs(assetsByColorScheme, destPath, !colorScheme.includes('mixed') || doNotRemoveSuffix);
```

maybe something like this? and then you can ignore my first comment.

Reply:
Ok so I've voted for your previous check.  For ease of reading (for me :). Both work and do what is intended.  Let me know if that is ok?  Thx.